src/EnergyPlus/WeatherManager.cc

```diff
@@ -2532,9 +2526,9 @@ namespace WeatherManager {
         OutDewPointTemp = TodayOutDewPointTemp(TimeStep, HourOfDay);
         if (EMSOutDewPointTempOverrideOn) OutDewPointTemp = EMSOutDewPointTempOverrideValue;
         OutRelHum = TodayOutRelHum(TimeStep, HourOfDay);
-        OutRelHumValue = OutRelHum / 100.0;
+        OutRelHumValue = OutRelHum * 1.0e-2;
```

Comment:
These changes are introducing tiny diffs. Let's not introduce that as part of this PR. This can be explored as a separate PR after I/O freeze.

Reply:
Done.